recipes/whatsapp/service.css

```diff
@@ -13,6 +13,6 @@
 }
 
 /* fix for blank spaces regarding view width on screens larger than 1080p */
-.app-wrapper-web ._1jJ70 {
+.app-wrapper-web ._1jJ70, .app-wrapper-web ._aigs {
```

Comment:
Is this needed?

Reply:
Removed old selector in [3f0b40f](https://github.com/ferdium/ferdium-recipes/pull/527/commits/3f0b40f128424b5c656724d2a875f45631cc3468)